auditor/evaluation/discriminative.py

```diff
@@ -90,6 +91,8 @@ def evaluate(
             f'Started model evaluation with perturbation type '
             f'{self.perturbed_dataset.perturbation_type}'
         )
+        progress_bar = ProgressLogger(total_steps=min(len(self.perturbed_dataset.data), len(self.perturbed_dataset.metadata)), description="Starting Model Evaluation")
```

Comment:
To get past lint check can you pass the arguments in the next line.

`./auditor/evaluation/discriminative.py:94:81: E501 line too long (167 > 80 characters)`